SPECS/kpatch/kpatch.spec

```diff
@@ -0,0 +1,74 @@
+Summary:        Kpatch tooling
+Name:           kpatch
+Version:        0.9.6
+Release:        1%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://github.com/dynup/kpatch
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildRequires:  binutils
+BuildRequires:  dnf-utils
+BuildRequires:  elfutils
+BuildRequires:  elfutils-devel
+BuildRequires:  gcc
+BuildRequires:  glibc-devel
+BuildRequires:  kernel-headers
+
+%description
+Kpatch is a Linux dynamic kernel patching infrastructure which allows you to patch
+a running kernel without rebooting or restarting any processes.
+It enables sysadmins to apply critical security patches to the kernel immediately,
+without having to wait for long-running tasks to complete, for users to log off, or
+for scheduled reboot windows.
+It gives more control over uptime without sacrificing security or stability.
+
+%package devel
+Summary:        Tools for building livepatches with kpatch.
+Group:          Development/Tools
+
+Requires:       numactl-devel
+Requires:       openssl
+Requires:       patch
+Requires:       wget
+
+%description devel
+%{summary}
+
+%prep
+%autosetup
+
+%build
+%make_build
+
+%install
+%make_install PREFIX=%{_prefix}
+
+# We don't need to install manuals.
+rm -rf %{buildroot}%{_mandir}
+
```

Comment:
Can we add a check section for this package? Upstream seems to have [some unit tests available](https://github.com/dynup/kpatch/blob/master/.github/workflows/unit.yml).

Reply:
I tried but tests' dependency on ShellCheck make the enabling the tests way out of scope, unfortunately. We don't have that tool and it depends on a bunch of packages we don't have as well. Didn't want to go down that rabbit hole.